.github/workflows/github.yml

```diff
@@ -384,3 +384,28 @@ jobs:
         name: ${{ env.VCMI_PACKAGE_FILE_NAME }}
         path: |
           ${{ env.ANDROID_APK_PATH }}
+
+
+  deploy-src:
```

Comment:
should be run only when making release, so something like

    if: always() && github.ref == 'refs/heads/master'

Reply:
Should be fixed now